ambari-server/src/main/java/org/apache/ambari/server/upgrade/UpgradeCatalog275.java

```diff
@@ -17,16 +17,27 @@
  */
 package org.apache.ambari.server.upgrade;
 
+import static org.apache.ambari.server.utils.CustomStringUtils.deleteSubstring;
```

Comment:
The same configuration come with the previous Grafana version included 2.7.5, but then - as a bug the config file updating was not done. This caused problem if one upgraded from 2.6.x. Now, that is the reason that code part is placed here, because it should have been added here at the previous Grafana upgrade.